app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -627,37 +627,63 @@ public boolean onPrepareActionMode(ActionMode mode, Menu menu) {
 
         @Override
         public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
-            int nSelected;
+            int nSelected = getSelectedItems().size();
             switch (item.getItemId()) {
                 case R.id.menu_item_delete:
                     AlertDialog.Builder d = new AlertDialog.Builder(context);
-                    d.setMessage("Are you sure ?").
-                            setCancelable(false).
-                            setPositiveButton("Delete", new DialogInterface.OnClickListener() {
-                                @Override
-                                public void onClick(DialogInterface dialog, int which) {
-
-                                    for (int i = getSelectedItems().size() - 1; i >= 0; i--) {
-                                        deleteMessage(getSelectedItems().get(i));
+                    if (nSelected == 1) {
```

Comment:
why snackbar? In most of the apps Toast is being used to display such type of informations.